docs/pipelines/index.md

```diff
@@ -45,15 +44,16 @@ Inputs with just a step name will be assumed to be `step.outputs`.
 
 The default payloads for Pipelines is the V2 protocol which requires named tensors as inputs and outputs from a model. If you require just certain tensors from a model you can reference those in the inputs, e.g. `mymodel.outputs.t1` will reference the tensor `t1` from the model `mymodel`.
 
-For the specification of the [V2 protocol](../apis/inference/index.md).
+For the specification of the [V2 protocol](broken-reference).
```

Comment:
isn't this now in `docs/inference/v2.md`?

Reply:
This issue has been fixed.